src/test/java/hudson/plugins/sonar/SonarBuildWrapperTest.java

```diff
@@ -110,7 +110,10 @@ public void maskAuthToken() throws IOException, InterruptedException {
     addCredential(CREDENTIALSID, MYTOKEN);
     configureSonar(inst);
 
-    OutputStream os = wrapper.decorateLogger(mock(AbstractBuild.class), bos);
+    AbstractBuild<?, ?> mock = mock(AbstractBuild.class);
+    when(mock.getCharset()).thenCallRealMethod();
```

Comment:
most mockito changes from here on are to do with mockito returning null for unmocked methods